layers/meta-balena-compulab/recipes-support/cpu-governor/gov-switch.bb

```diff
@@ -0,0 +1,38 @@
+DESCRIPTION = "Resin custom INIT file"
```

Comment:
Can you try the schedutil governor and test your usecase? It should be what you are looking for and you could drop this script too

Reply:
The schedutil governor sets the CPU frequency to be proportional to the utilization/capacity ratio coming from the scheduler. Our issue is that the CPU experiences a rise in temperature because of the enclosure, and the system reboots.
Currently, the scaling_governor is set to ondemand by default. We could change it to schedutil but we will still need this script to monitor the temperature and set the scaling_governor to powersave when the temperature goes above a critical threshold.